pkg/test/echo/response/instances.go

```diff
@@ -0,0 +1,55 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package response
+
+import (
+	"fmt"
+)
+
+// Instances is an ordered list of parsed response objects.
+type Instances []*Instance
```

Comment:
nit: do we need pointers here and ~everywhere else? I think these shouldn't be mutable and are not large so not sure pointer is needed?